cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -108,7 +108,7 @@ def alter_field(self, url, details, field, val):
         payload['enrollment_start'] = CourseDetailsViewTest.convert_datetime_to_iso(details.enrollment_start)
         payload['enrollment_end'] = CourseDetailsViewTest.convert_datetime_to_iso(details.enrollment_end)
         resp = self.client.ajax_post(url, payload)
-        self.compare_details_with_encoding(json.loads(resp.content), details.__dict__, field + str(val))
+        self.compare_details_with_encoding(json.loads(resp.content.decode('utf-8')), details.__dict__, field + str(val))
```

Comment:
@feanil `.decode('utf-8')` its repeatings 100s of time in code base. Single method would be useful for this situation pass the content and return the decoded data. At-least for tests it would be helpful.

Reply:
Given that the function would just be one line, I'm not sure how valuable it would be.  Wouldn't it just be 
```
def decode(content):
    return content.decode('utf-8')
```

It's not really saving us much I think.